Batch8CucumberSeleniumAPI/src/test/java/stepdefs/ui/SearchStepDefs.java

```diff
@@ -75,15 +75,19 @@ public void i_have_browser_opened_and_url_is_navigated() throws Exception {
 	public void i_search_for_product_as(String product) {
 		testContextUI.getCmnPageObjects().SetSearchTextBox(product);
 		testContextUI.getCmnPageObjects().ClickOnSearchButton();
-		scn.write("Search was sucessfull");
-		
+		scn.write("Search was sucessfull");	
 	}
 
 	@Then("product list should appear pertaining to the product search as {string}")
 	public void product_list_should_appear_pertaining_to_the_product_search_as(String productName) {
 		testContextUI.getSearchPageObjects().ValidateProductList(productName);
 	}
-	
+
+	@Then("Search result should displayed {string}")
+	public void search_result_should_displayed_Try_Checking_Your_Spelling_Or_Use_More_General_Term(String productName) {
+		testContextUI.getSearchPageObjects().ValidateInvalidateProductMessage();
```

Comment:
Pass the argument to the method and validate that:
it should look like this:

`testContextUI.getSearchPageObjects().ValidateInvalidateProductMessage(productName);`